README.md

```diff
@@ -12,7 +12,7 @@ Configure pom.xml, like this:
     <dependency>
         <groupId>org.springframework.cloud</groupId>
         <artifactId>spring-cloud-config-server-mongodb</artifactId>
-        <version>0.0.3.BUILD-SNAPSHOT</version>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
```

Comment:
I think we're not ready for a 1.0.0 yet. Let's have this as 0.0.4.BUILD-SNAPSHOT.